src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -2,10 +2,7 @@
 
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<T> {
```

Comment:
```suggestion
public interface MachineProducer<T extends Machine> {
```
because we do not want to allow some `String` or anything that is not a `Machine` here